tasks/main-tang.yml

```diff
@@ -26,6 +26,10 @@
   when: (nbde_server_fetch_keys | bool) or (nbde_server_deploy_keys | bool)
   include_tasks: tang-key-management.yml
 
+- name: Open port in firewalld
+  when: (nbde_server_port != "80") or (nbde_server_firewall_zone != "public")
```

Comment:
`(nbde_server_port | int != 80)`

`nbde_server_port` can be a string or an int - if `nbde_server_port` is an int e.g. `nbde_server_port: 80` then `80` is not equal to `"80"`

Reply:
I've had so much problems with mode and always quoting it, so for me by default all numbers should be quoted. But sure I can fix that also. 

https://github.com/ansible/ansible/issues/9196

Reply:
> I've had so much problems with mode and always quoting it, so for me by default all numbers should be quoted. But sure I can fix that also.

If you want to keep everything a string, you can do that too:
`(nbde_server_port | string != "80")`

But in the examples you added, you are using `int` instead of `string` e.g. https://github.com/linux-system-roles/nbde_server/pull/38/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R37 , https://github.com/linux-system-roles/nbde_server/pull/38/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R124, etc.

So whatever you decide to do, please be consistent

> 
> [ansible/ansible#9196](https://github.com/ansible/ansible/issues/9196)